src/Plugin.ts

```diff
@@ -57,9 +58,10 @@ export class Plugin {
     browser: Cypress.Browser,
     args: string[]
   ): string[] {
+    this.browser = browser;
     if (!this.isSupportedBrowser(browser)) {
```

Comment:
To avoid duplication in methods below and need to add new field solely for calling `isSupportedBrowser`, I suggest assigning the result of the execution to an instance property like this:
```suggestion
    this.installed = this.isSupportedBrowser(browser);
    if (!this.installed) {
```

Reply:
👍 Updated here:
https://github.com/NeuraLegion/cypress-har-generator/compare/2c2f3cae78afa87f9d98c932becca649a7405137..22e6cdf7d1f2a8ab4cf9bf301fb8f321d98b1ce3

To accommodate this change, I also had to move the precondition check to the beginning of `recordHar` and add a missing `ensureBrowserFlags` call to one of the test cases.